### PR TITLE
User/le2tang/dispatcher demo

### PR DESCRIPTION
**Pull request by le2tang**
Add field to control which phase is started in the mission plan.
Tested using dummy tasks, needs full run through test

### CODE REVIEW

File: nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py [hunk 1]
```diff
@@ -31,20 +31,30 @@ def __init__(self):
...
-        self.phases = queue.Queue()
-
     def go_to_patient_phase(self):
         rospy.loginfo("Nightingale Mission Planner going to patient")
         # Assume door is open
         # TODO: first go to doorside then bedside when door opening added
 
         # update to driving screen
-        task_reponse = self.send_interface_request_task.execute(RobotStatus.DRIVING)
+        task_response = self.send_interface_request_task.execute(RobotStatus.DRIVING)
```
Comment (FelixKHWong) [inline nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py:60]: why is this a change lol

**Review by FelixKHWong — DISMISSED** [review]

File: nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py (hunk 1 above)
Comment (le2tang) [reply nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py:60]: I needed to run a full test, and there was another instance where this was spelled wrong that could prevent a full test, so i did a find and replace.

Comment (FelixKHWong) [reply nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py:60]: oh I did not see that nice

**Review by bdschnapp — COMMENTED** [review]
I would like to see this tested IRL before approving, but otherwise it looks good to me

**Pull request merged**